test/unit/tags/breadcrumbs.ts

```diff
@@ -23,55 +23,62 @@ suite('gb-breadcrumbs', Breadcrumbs, ({
 
   describe('clearRefinements()', () => {
     it('should update refinements with empty array', () => {
-      const updateRefinements = stub(tag(), 'updateRefinements');
-
+      spy = tag().update = sinon.spy((obj) => expect(obj).to.eql({ selected: [] }));
```

Comment:
So this would become
`const update = tag().update = spy()`

then I'd end with the expectation:

`expect(update.calledWith({ selected: [] })).to.be.true;`